tests/e2e/README.md

```diff
@@ -75,12 +75,29 @@ kubectl create clusterrolebinding myname-cluster-admin-binding    --clusterrole=
 
 ## Step 4: Export test script variables
 
-**Option 1:** Build your own images.
+**Option 1:** Build your own images and use an in-cluster registry.
+# This allows you to set up a private docker registry within the cluster without connection to a remote one.
+# Please fill in the ip address of the registry in <registry-ip> of HUB value, 5000 is the exposed port of the registry.
+# For local registry in vagrant-kubernetes-istio settings, registry ip should be 10.10.0.2
```

Comment:
And have you verified that this option works in GKE?

Reply:
Not yet, but I tried that on Bluemix Container Service. I had no problem in pushing the images to it, but failed on pulling since insecure registry is not enabled. This should be the same case for GKE as well. I found ways to do that already. Will update the README later.

Reply:
Sweet thanks.

Reply:
Did you try any comparison between local registry and GCR on GKE? If it is really faster, we should probably change all prow tests to use that.

Also, now that I had updated this README.md pretty significantly, shall we make the update in the local VM option instead?

Reply:
I can try to run it with GCR, will tell you the time difference later.
I made some updates in the README.md under localregistry directory. I think that looks like a more general way. What do you think?

Reply:
There are way too much distributed information, and I doubt many people knows about localregistry/README.md.

I would like to make our local VM options (both minikube and vagrant) use the localregistry in cluster option by default. We should update the script and docs to reflect that.

And it probably makes more sense to move the content of localregistry readme to the local testing doc (or a separate page in the same folder), and point localregistry/README.md back to the e2e/README.

WDYT?

Reply:
I think this makes sense. Will make changes.